### ISSUE TITLE
Release 1.1.0 has wrong version number

### ISSUE THREAD

**Issue by bastelfreak** [camptocamp/prometheus-puppetdb-exporter#9]
Hi people,

while implementing your latest release into puppet (https://github.com/voxpupuli/puppet-prometheus/pull/513) I noticed that the version number wasn't updated:

```
$ ./prometheus-puppetdb-exporter-1.1.0.linux-amd64/prometheus-puppetdb-exporter --version
INFO[0000] PuppetDB Metrics Exporter 1.0.0-5-gd6c5800    build date: 2020-12-03    sha1: d6c5800fcd9c2c0eece9ab9bdaf26ad6556e1220    Go: go1.15.4 
```

could you do a 1.1.1 release with correct version reporting?

**Comment by cryptobioz** [comment]
I rebuilt the binary and published a new archive for the [release 1.1.0](https://github.com/camptocamp/prometheus-puppetdb-exporter/releases/tag/1.1.0). It should fix this issue.

**Comment by raphink** [comment]
@cryptobioz can this issue be closed then?
